### DISCUSSION TITLE
Release 0.6.6 (1.21.10.22)

### DISCUSSION THREAD

**Discussion by m-brisch** [Mojang/minecraft-editor#180] [Announcements]
_[Read the previous release notes here](https://github.com/Mojang/minecraft-editor/discussions/179)_  

---    

## Summary  

Hello and happy preview day 🥳! We are now on 0.6.6 (1.21.10.22), as of Wednesday, June 5th, 2024. 

As a reminder of Editor API support:   

- Editor Extensions [overview article](https://learn.microsoft.com/en-us/minecraft/creator/documents/editorextensionsintroduction) and [API reference documentation](https://aka.ms/EditorAPI) on learn.microsoft.com  
- [Starter Kit](https://github.com/Mojang/minecraft-editor-extension-starter-kit): Contains the compiler tools and various libraries required to get started making Editor Extensions  
- [Samples](https://github.com/Mojang/minecraft-editor-extension-samples) created by our team to jump-start your own Extension creations  

Additionally, if you're new to the Editor GitHub this week, we encourage you to join the Bedrock Add-Ons Discord [#Editor channel](https://discord.gg/TZnQKFwGpt)!  

---

## Change Log   

- Removed Keybinding for ‘D’ from Paste Preview which blocked movement right. Fixed Distance mode can be selected using the side-panel
- Fixed missing CTRL+X key binding in Editor
- Expanded the Block Palette Service API to allow management of Block Palettes and Block Palette Items
- Added radial drag to number input fields, allowing value changes with mouse drag movement
- Introduced a new script brush manager method getBrushVolume which, given a set of brush pipeline operations, builds an absolute compound block volume for the brush
- Fixed bug where small movements in the brush tool caused incorrect cursor tracking

---   

Thank you for being here! You're the best! ❤️

**Comment by RandyCHS** [comment]
I'm just starting to play around with this tool.  I'm using the Education Edition of Minecraft (MEE) in my high school computer science classes.  I can't seem to find a way to load this editor in the Preview Edition of MEE (1.20.70.0), but I can load it in the regular Preview Edition of Bedrock Minecraft (1.21.10.23).  When I try to load any of my worlds created with this editor into MEE-Preview, I get a this message, 'A new version of the game has saved this world.  It cannot be loaded'.  Any possible workarounds for this?

**Comment by m-brisch** [comment]
Hello @RandyCHS! Sorry for the delay in answering here. Try using https://chunker.app/ to convert the world and see if that works.

**Reply by RandyCHS** [reply]
@m-brisch Thanks for getting back to me and for the link to Chunker.  First time I tried converting my edited world (created in Minecraft Preview 1.21.1.23) using the 1.21.0 setting.  That converted version gave me the same error message in the new the Preview Edition of MEE (1.21.01.0). Then I converted the same edited world using the 1.20.80 setting and that would load into the new the Preview Edition of MEE.  So it seems even the 1.21.0 conversion is still 'newer' that the 1.21.01.0 version of MEE. 🤷‍♂️️ Nonetheless, at least I can now create worlds for MEE. Thanks. 🙏
